src/main/java/baritone/process/FarmProcess.java

```diff
@@ -225,6 +269,7 @@ public PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
                 if (ctx.isLookingAt(pos)) {
                     baritone.getInputOverrideHandler().setInputForceState(Input.CLICK_LEFT, true);
                 }
+
```

Comment:
What are all these blank lines?

Reply:
The formatting of this code is very bad I'm srorry